File: Linguagem C/1037.c
#include <stdio.h>
float entrada = 0;

int main(){
	scanf("%f",&entrada);
	
	if(entrada >= 0 && entrada <= 25){
		printf("Intervalo [0,25]\n");
	}else if(entrada >= 25 && entrada <= 50){
		printf("Intervalo (25,50]\n");
	}else if(entrada >= 50 && entrada <= 75){
		printf("Intervalo (50,75]\n");
	}else if(entrada >= 75 && entrada <= 100){
		printf("Intervalo (75,100]\n");
	}else{
		printf("Fora de intervalo\n");
	}
	
	
	return 0;
}

File: Linguagem Java/1037.java
package pkg1037;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
        double n = input.nextDouble();
        
        if(n >= 0 && n <= 25){
            System.out.println("Intervalo [0,25]");
        }else if(n >= 25 && n <= 50){
            System.out.println("Intervalo (25,50]");
        }else if(n >= 50 && n <= 75){
            System.out.println("Intervalo [50,75]");
        }else if(n >= 75 && n <= 100){
            System.out.println("Intervalo (75,100]");
        }else{
            System.out.println("Fora de intervalo");
        }
        
        
    }
    
}
